src/components/activities/patientDetailsActivity/PatientDetailsActivity.tsx

```diff
@@ -375,12 +379,26 @@ const PatientDetailsActivity: FunctionComponent<TProps> = ({
                       <Route path={`${path}/admissions`}>
                         <PatientDetailsContent
                           title="Admissions"
+                          permissions={[
+                            "admission.read",
+                            "admission.create",
+                            "admission.update",
+                          ]}
```

Comment:
I think that is more simple wrap each component with a Permission checker, don't you?